ml-agents/mlagents/trainers/torch/networks.py

```diff
@@ -252,7 +265,7 @@ def __init__(
         self.act_type = act_type
         self.act_size = act_size
         self.version_number = torch.nn.Parameter(torch.Tensor([2.0]))
```

Comment:
Have we decided on using Parameter with require_grad or register_buffer?

Reply:
I'd go with register_buffer since it's the "PyTorch way". With that said, this one is there purely for ONNX export - would that cause an issue? Maybe we should add a requires_grad = False for this one.

Reply:
That's possible. Make it requires_grad=False is good enough then.

Reply:
I don't think these need to be saved at all actually, since they're not affected by the training process - maybe we can just set them to bare Tensors?